civihr_default_theme/scss/base/_variables.scss

```diff
@@ -34,6 +34,7 @@ $gray                 : #767b8e; // main text/lighter, menu item bg hover, foote
 $gray-dark            : #555862; // menu bg, default: lighten(#000, 20%) !default
 $gray-darker          : #4c4c5d; // text darker, default: lighten(#000, 13.5%) !default; // #222
 $crm_gray-darker-alt  : #404454; // page title
+$crm_gray-border      : #777C8F; // page title
```

Comment:
Removed $crm_gray-border completely and using **$grey** instead, as the color difference is not significant.